schemas/+acquisition/Session.m

```diff
@@ -10,9 +10,13 @@
 stimulus_commit = ''        : varchar(64)                   # git hash for the version of the function
 stimulus_set                : tinyint                       # an integer that describes a particular set of stimuli in a trial
 session_performance         : float                         # percentage correct on this session
+num_trials=-1               : int(11)                       # Number of trials for the session
+num_trials_try=null         : tinyblob                      # Accumulative number of trials for each try of the session
 session_narrative = ''      : varchar(512)                  # descriptive string
 session_protocol=null       : varchar(255)                  # function and parameters to generate the stimulus
 session_code_version=null   : blob                          # code version of the stimulus, usually two numbers "maze_version=2.1, code_version = 4.0". In the future: a githash?
+is_bad_session=0            : boolean                       # Flag that indicates if this session had any issues
```

Comment:
For old sessions where we don't quite know whether they are bad, would 'null' make more sense or we treat them as good?

Reply:
I think by default sessions are "good". 
It makes more sense to me to just flag bad sessions than go and flag every session as good or bad.